examples/webgl_gpgpu_water.html

```diff
@@ -122,6 +126,67 @@
 			}
 
 		</script>
+		
+		<!-- This is a 'compute shader' to calculate the current volume of water at the center -->
```

Comment:
You mean current _level_ of the water.

There are multiple occurrences of the word 'volume' in this PR.

Reply:
Fixed.